kolibri/plugins/learn/assets/src/vue/index.vue

```diff
@@ -37,12 +52,44 @@
 
 <script>
 
+  const getters = require('../state/getters');
+  const constants = require('../state/constants');
+  const store = require('../state/store');
+  const PageNames = constants.PageNames;
+
   module.exports = {
+    mixins: [constants], // makes constants available in $options
```

Comment:
Yeah, I dislike this also. It's not really a prop, though - it's something that should just be required - the difficult part was getting required stuff into the template.

In other places I compute the link as a computed property, and I think that's a bit easier to understand. E.g. https://github.com/indirectlylit/kolibri/blob/63d0cc453b212970e2fd44f0a87cf3f42522bde5/kolibri/plugins/learn/assets/src/vue/content-card/index.vue#L3

I can work on removing all the `$options` as a subsequent task if that's cool